packages/plugin-core/src/test/suite-integ/DoctorCommand.test.ts

```diff
@@ -266,3 +270,37 @@ suite("notes", function () {
 //       setupDendronWorkspace(root.name, ctx);
 //     });
 //   });
+
+suite("CREATE_MISSING_LINKED_NOTES", function () {
+  let ctx: vscode.ExtensionContext;
+
+  ctx = setupBeforeAfter(this, {
+    afterHook: () => {
+      sinon.restore();
+    },
+  });
+
+  test.only("basic", (done) => {
+    runLegacySingleWorkspaceTest({
+      ctx,
+      postSetupHook: ENGINE_HOOKS.setupBasic,
+      onInit: async ({ wsRoot, vaults }) => {
+        const vault = vaults[0];
+        await NoteTestUtilsV4.createNote({
+          fname: "real",
+          body: "[[real.fake]]\n",
+          vault: vault,
+          wsRoot,
+        });
+        const cmd = new DoctorCommand();
+        sinon.stub(cmd, "gatherInputs").returns(
+          Promise.resolve({
+            action: DoctorActions.CREATE_MISSING_LINKED_NOTES,
+          })
+        );
+        await cmd.run();
+        done();
```

Comment:
don't forget to check if the note is created

Reply:
ah yeah. this test file isn't complete 😄  I pushed it with the code mid way